app/controllers/mixins/start_url.rb

```diff
@@ -41,7 +36,6 @@ def start_page_options
 
   def start_page_allowed?(start_page)
     return false if STORAGE_START_PAGES.include?(start_page)
```

Comment:
I think it prevents "Storage / Storage Managers" from appearing under EVM -> My Settings -> Visual -> "Start Page" dropdown.
Aha, it would not be there anyway because it's not in `MiqShortcut.start_pages` because
https://github.com/manageiq/manageiq/blob/master/db/fixtures/miq_shortcuts.yml#L197-L200
doesn't set `:startup: true`.

Anyway I think STORAGE_START_PAGES & this line should be dropped like this PR did for CONTAINERS_START_PAGES.  Before #204 the logic was
```
    return false if STORAGE_START_PAGES.include?(start_page) && !::Settings.product.storage
```
which "allowed" it to be seen conditionally on the setting (if only it had `:startup: true`), until the setting disappeared without revising this file.